config/config-package.py

```diff
@@ -32,6 +32,16 @@
 
 DOCKER_IMAGE = 'python:3.11-bullseye'
 
+GHA_DEFAULT_BRANCH = "main"
```

Comment:
If we go by the `ref` suggestion, then:

```suggestion
GHA_DEFAULT_REF = "main"
```